src/main/java/Crystal.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Crystal {
+    public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
+        System.out.println("____________________________________________________________\n"
+                + "Hola! I'm Crystal.\n"
+                + "What can I do for you today?\n"
+                + "____________________________________________________________\n");
+        line = in.nextLine();
+
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                System.out.println("____________________________________________________________");
+                for (int i = 0; i < tasks.size(); i++) {
+                    System.out.println((i + 1) + ". " + tasks.get(i));
+                }
+                System.out.println("____________________________________________________________");
+            } else if (line.startsWith("todo")){
+                String description = line.substring(5).trim();
```

Comment:
Perhaps the number '5' here could be turned into a constant so that it isn't a "magic number", as with the case for other constant numbers in this code.